.travis.yml

```diff
@@ -25,6 +25,12 @@ before_install:
   - ./runflake8
   - ./runisort
 
+  # Install JSlint:
+  - npm install --save jslint -g
+
```

Comment:
remove blank line, also remove the comments because they are not necessary (it's pretty obvious that we are installing jslint and we are running a jslint check), also remove the blank line after `- ./runisort`